CONTRIBUTING.md

```diff
@@ -22,9 +51,9 @@ When you submit a pull request it can be reviewed and commented on by anyone.
 For the code to be accepted into SunPy it must meet the following criteria:
```

Comment:
Add "Also, *everyone's* contributions gets reviewed, and no-one's code is accepted in to SunPy without being reviewed.  We do this to try to maintain a consistent coding style and high quality code that is easy to read and understand".